packages/babel-types/scripts/generators/flow.js

```diff
@@ -99,6 +99,14 @@ for (const type in t.NODE_FIELDS) {
         ", "
       )}): ${NODE_PREFIX}${type};`
     );
+  } else {
+    const functionName = toFunctionName(type);
+    lines.push(
+      `declare function _${functionName}(${args.join(
```

Comment:
Maybe this should be `export declare function ${type}`? They always start with an uppercase letter (thus are valid identifiers), and it would match https://github.com/babel/babel/blob/6467cd1e27ffe2a3a1cfe97bc8c67b407cb1e7c4/packages/babel-types/src/builders/generated/index.js

Reply:
@nicolo-ribaudo what do you think?